### ISSUE TITLE
System accessibility settings which are set to non-normal sizes interfere with keyboard and touch logic

### ISSUE THREAD

**Issue by powline99** [florisboard/florisboard#1953]
#### Short description
When I go to "tap" type (not swipe) on multiple apps using Android / Lineage Coral Phone with Floris 0.3.16 , Android 11, Pixel 4 XL the space bar sticks causing me to have to hit it multiple times (occasionally). This also misspells words. I tried to turn off new spacebar functionality (turned off all "spacebar gestures"). 
Could it have to do with new functionality of the space bar to navigate or do special functions?? 
It does not do this on stock droid keyboard. 
It did not do this on 2 previous versions (Floris stable) before spacebar functionality was added. 

Update:
To Add to Above ^^ . I use Accessibility Large Font Size, and Large Display Size. Might be part of bug??

#### Steps to reproduce
1. Go to App like Brave Browser, Chrome, IG, FB, Notepad, etc...
2. Type
3. Swipe typing does not do this, just tap typing. 
4. Hit space bar multiple times if the words run together

#### Environment information
- FlorisBoard Version: 0.3.16
- Install Source: Fdroid Stable  Ver
- Device: Pixel 4 XL
- Android: 11
- Lineage Coral

**Comment by patrickgold** [comment]
Thanks for bug report!

I currently don't really know what could cause the problem, besides maybe the accessibility large font and display size. Would it be possible for you to test this out by setting these two accessibility options back to normal for a short period and typing a few sentences with it?

**Comment by powline99** [comment]
> Thanks for bug report!
> 
> I currently don't really know what could cause the problem, besides maybe the accessibility large font and display size. Would it be possible for you to test this out by setting these two accessibility options back to normal for a short period and typing a few sentences with it?

Yes - Accessibility Large Font Size, and Large Display Size does seem to cause a issue.

**Comment by patrickgold** [comment]
Ok thanks for testing it out - will rename this issue then.
